### DISCUSSION TITLE
添加检测是否为 服务端mod

### DISCUSSION THREAD

**Discussion by RaukaRur** [Meloong-Git/PCL#2687] [提问] [answered]
**描述**
在mod列表中添加按钮 查询是否为服务端mod
**原因**
有些整合包并没有给出官方服务端 自己从整合包里面扣mod去mcmods搜是否为服务端mod 实在是麻烦
PCL能不能起到一个对接mcmods来自动查询是否为服务端mod

**Comment by WForst-Breeze** [comment, marked as answer]
真的可以检测出来吗（）请前往 Issue 页面提交新功能提案

edit：#2689 已拒绝
